pkgs/by-name/ae/aegisub/package.nix

```diff
@@ -167,5 +144,7 @@ stdenv.mkDerivation (finalAttrs: {
     mainProgram = "aegisub";
     maintainers = with lib.maintainers; [ wegank ];
     platforms = lib.platforms.unix;
+    # No native wxWidgets and VapourSynth is broken.
```

Comment:
I'm unable to test VapourSynth on darwin but if its broken maybe also add `badPlatforms` to the VapourSynth package.

Reply:
Oh nvm I should have looked at the whole file before commenting